gctscript/vm/vm_types.go

```diff
@@ -0,0 +1,59 @@
+package vm
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/d5/tengo/script"
+	"github.com/gofrs/uuid"
+)
+
+const (
+	AuditEventID = "gctscript"
+	// DefaultTimeoutValue default timeout value for virtual machines
+	DefaultTimeoutValue = 1 * time.Minute
```

Comment:
rm 1 *

Reply:
Lowered this down to 30 seconds for timeout as its more than enough for most